schematic/utils/validate_utils.py

```diff
@@ -3,26 +3,33 @@
 from jsonschema import validate
 
 from schematic import CONFIG
+from schematic.loader import Loader, InvalidResourceError
+
+# call Loader() and pass `schematic`, which is the global package namespace
+loader = Loader('schematic', prefix='etc')
```

Comment:
It seems like you are creating a loader for every submodule. I'm still unsure why we need a loader (as opposed to a single function, like the one below), but if this pattern is useful, then you might want to consider doing like what I did for `CONFIG` and instantiate one loader as a module-level constant. 

```
def get_bundled_file(filename):
    return pkg_resources.resource_stream(__name__, f'etc/{filename}')
```

Reply:
Awesome, addressed this in a new commit.

Reasoning for adopting `Loader()` pattern detailed in above comment.